modules/ROOT/pages/errors/gql-errors/22NC3.adoc

```diff
@@ -0,0 +1,44 @@
+= 22NC3
+
+== Status description
+error: data exception - relationship element type is ineffectual. The relationship element type `{ <<relType>> }` must define a source or destination, or at least one property type.
```

Comment:
In general I like the new version, just one small thing
```suggestion
error: data exception - relationship element type is ineffectual. The relationship element type `{ <<relType>> }` must define a source, destination, or at least one property type.
```
could probably just do this then?